USAGE.md

```diff
@@ -1,370 +1,707 @@
+}
+
+collection = Collection(f"https://cti-taxii.mitre.org/stix/collections/{collections['enterprise_attack']}/")
+src = TAXIICollectionSource(collection)
 ```
 
-Once this filter is defined, you can pass it to the DataSource `query` function in order to actually query the data:
+### Access from Github via requests
+Users can alternatively access the data from MITRE/CTI using HTTP requests, and load the resulting content into a MemoryStore. 
+While typically the TAXII method is more desirable for "live" access, this method can be useful if you want to 
+access data on a branch of the MITRE/CTI repo (the TAXII server only holds the master branch) or in the case of a TAXII server outage.
 
 ```python
-techniques = fs.query([filt])
+import requests
+import stix2
+
+def get_data_from_branch(domain, branch="master"):
+    """get the ATT&CK STIX data from MITRE/CTI. Domain should be 'enterprise-attack', 'mobile-attack' or 'pre-attack'. Branch should typically be master."""
+    stix_json = requests.get(f"https://raw.githubusercontent.com/mitre/cti/{branch}/{domain}/{domain}.json").json()
+    return stix2.MemoryStore(stix_data=stix_json["objects"])
+
+src = get_data_from_branch("enterprise-attack")
 ```
 
-Notice that the `query` function takes a **list** of filters.  These filters are logically AND'd together during the query. As of this writing, `allow_custom` must be set to `True` in order to query ATT&CK data. This is because the ATT&CK data uses several custom properties which are not part of the STIX 2.0 specification (`x_mitre_platforms`, `x_mitre_contributors`, etc). **UPDATE(8-March-2018)**: Python STIX-2 has moved 'allow_custom' to be set at the DataStore/Source level, not within each API call (i.e. get(), query() etc...). Also, all DataStores/Sources by default now set 'allow_custom' to True.
+## Accessing a specific version of ATT&CK
 
-**For the remaining examples, these imports and the FileSystemStore initialization will be omitted.**
+ATT&CK versions are tracked on the MITRE/CTI repo using [tags](https://github.com/mitre/cti/tags). Tags prefixed with `ATT&CK-v` correspond to ATT&CK versions and tags prefixed with `CAPEC-v` correspond to CAPEC versions. You can find more information about ATT&CK versions on the [versions of ATT&CK page](https://attack.mitre.org/resources/versions/) on the ATT&CK website. 
 
-### Get all Software
-Since ATT&CK software can either be classified as a `tool` or `malware` in STIX, you must query for both of them in order to find all software. The library's `query` function does not have the capability to do logical OR, so two separate queries must be performed.  The results are merged together into one list.
+In addition to checking out the repo under the tag for a given version or downloading the STIX from github using your browser, you can also use a variation on the requests method [above](#access-from-github-via-requests) to access using requests in Python:
```

Comment:
The last part seems like it's missing something. Maybe:

...to access _a particular version_ using requests in Python:

Reply:
Or just end the sentence at "you can also use a variation on the requests method above.", since it's sort of redundant.

Reply:
Changed to:

> ... to access a particular version of ATT&CK: